bioconda_utils/docker_utils.py

```diff
@@ -370,7 +370,14 @@ def _build_image(self):
         #  `docker version` command (note the missing dashes) is not consistent
         # between different docker versions. The --version string is the same
         # for docker 1.6.2 and 1.12.6
-        s = sp.check_output(["docker", "--version"]).decode()
+        try:
+            s = sp.check_output(["docker", "--version"]).decode()
+        except FileNotFoundError as e:
+            logger.error('DOCKER FAILED: Error checking docker version, is it installed?')
+            raise e
+        except sp.CalledProcessError as e:
+            logger.error('DOCKER FAILED: Error checking docker version.')
+            raise e
```

Comment:
Can you remove the ` as e` / ` e` in the `except` and `raise` expressions, resp.? That way we get an unaltered stack trace.